model/project_ref_test.go

```diff
@@ -786,29 +786,6 @@ func TestDefaultRepoBySection(t *testing.T) {
 	}
 }
 
-func TestGroupProjectsByRepo(t *testing.T) {
```

Comment:
Can we make a new test (or adapt this one) for GroupProjects?

Reply:
There is a graphql test for it 
https://github.com/evergreen-ci/evergreen/blob/1e8af407957a6e03874fb1c101009e681f0f9166/graphql/tests/project/results.json